services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/AlluClient.kt

```diff
@@ -206,6 +209,18 @@ class AlluClient(
         }
     }
 
+    fun operationalCondition(alluApplicationId: Int, operationalConditionDate: LocalDate) {
+        put(
+                "excavationannouncements/$alluApplicationId/operationalcondition",
+                operationalConditionDate.atStartOfDay(TZ_UTC).toJsonString())
+            .bodyToMono(Void::class.java)
+            .timeout(defaultTimeout)
+            .doOnError(WebClientResponseException::class.java) {
+                logError("Error canceling application in Allu", it)
```

Comment:
Copy-paste mistake with "cancelling"?

Reply:
Yes, thank you. Wrote a proper error message.